### PR TITLE
Entities for Questions added

### PR DESCRIPTION
**Pull request by jangjh0421**
added blank classes for MultipleChoiceQuestion and OpenEndedQuestion
edited Question abstract class

### CODE REVIEW

**Review by vivjd — CHANGES_REQUESTED** [review]
Hi Jae, I've reviewed your code. I think it's awesome but...
1. Could you add specific implementations within each concrete class? So the MCQs will have the a "Options" list that contains the possinble choices.
2. The "answer" for the MCQ should be an index within the Hashmap, so change the MCQ answer to a hashmap as well.

Thank you!

File: src/entity/MultipleChoiceQuestion.java [hunk 1]
```diff
@@ -1,5 +1,26 @@
 package entity;
 class MultipleChoiceQuestion extends Question {
+    private Map<int, String> answerOptions = new HashMap<>();
+    private int correctAnswer;
 
+    public OpenEndedQuestion(){}
```
Comment (vivjd) [inline src/entity/MultipleChoiceQuestion.java:6]: This constructor should be MultipleChoiceQuestion right?

Comment (jangjh0421) [reply src/entity/MultipleChoiceQuestion.java:6]: oh yeah my bad will edit it right away thanks!!

File: src/entity/MultipleChoiceQuestion.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    public String getQuesion(){
+        this.getQuesion();
+    }
+    public String getanswerOptions() {
+        return answerOptions;
+    }
+    public void setanswerOptions(Map<int, String> ansOps) {
+        this.answerOptions = ansOps;
+    }
+    public String getCorrectAnswer() {
```
Comment (vivjd) [inline src/entity/MultipleChoiceQuestion.java:20]: Why is this method called getCorrectAnswer in this class but getAnswer in the other class? Would it be better to have it both as getCorrectAnswer?

**Review by vivjd — CHANGES_REQUESTED** [review]
I realized there may be a couple of more issues:
1. the setQuestion and getQuestion methods should be implemented like `this.question = question` and `return question` respectively?
2. We never actually added the question itself to the object, shouldn't we create a `String question` instance variable within each Question class?

**Review by vivjd — APPROVED** [review]
Looks good! Thanks

**Pull request merged**